doc/cephadm/install.rst

```diff
@@ -45,7 +45,7 @@ curl-based installation
   
   .. prompt:: bash #
 
-   curl --silent --remote-name --location https://github.com/ceph/ceph/raw/octopus/src/cephadm/cephadm
+   curl --silent --remote-name --location https://github.com/ceph/ceph/raw/pacific/src/cephadm/cephadm
```

Comment:
please avoid hardcoding the name of stable branch. please use
```rst
  .. prompt:: bash #
     :substitutions:

     curl --silent --remote-name --location \
       https://github.com/ceph/ceph/raw/|stable-release|/src/cephadm/cephadm
```

also, i'd suggest make this change on master. and backport it to pacific.

Reply:
looks like that change already exists in master. this should be backported to pacific #40118 instead of this pr